rtdata/languages/default

```diff
@@ -4331,6 +4383,11 @@ TP_WAVELET_OPACITYWL_TOOLTIP;Modify the final local contrast at the end of the w
 TP_WAVELET_PASTEL;Pastel chroma
 TP_WAVELET_PROC;Process
 TP_WAVELET_PROTAB;Protection
+TP_WAVELET_PYRWAVTRC;Contrast profile
+TP_WAVELET_PYRWAVTRC_TOOLTIP;The 'Contrast profile' slider lets you select a series of presets that will significantly modify the effect of the “Variable contrast” curve. The presets range from least to most aggressive in ascending order.\n\nFor the upper preset levels, the visibility of the result in the preview window will depend on the image size and the system configuration. Increase the preview window size if this is the case.
+TP_WAVELET_PYRWAVTRC_CURVE_TOOLTIP;The Variable contrast curve takes into account the information provided by the presets. The mid-point of the x-axis corresponds to the predominant contrast value, irrespective of the detail level and the selected preset. The maximum point, to the right of the x-axis, corresponds to the maximum contrast value. It is recommended to reduce the variable contrast beyond the mid-point.
+TP_WAVELET_PYRWAVTRC_RESID_TOOLTIP;Residual contrast, adjusts the residual image after decomposition to compensate for excessive effects linked to the action of the curve and the presets (relief effect, contrast differences between structure and uniform areas).
+TP_WAVELET_PYRWAVTRC_SIGMA_TOOLTIP;Attenuation response, distributes the action between the low, medium and high values of the “Variable contrast” curve by widening or narrowing the width of the curve.
```

Comment:
This can be more concise.

```suggestion
TP_WAVELET_PYRWAVTRC_SIGMA_TOOLTIP;Stretches the width of the "Variable contrast" curve. A value of 1 means no stretching.
```

Reply:
Again, I don't really know what to answer... The usual toolip is the one written below, but in other cases it is integrated into "wavelets" tools... Here it is the same tool, the same function but "hidden", we don't speak of wavelet (an engineer's thing)
I wanted to make a different annotation (so as not to scare), but it is not the exact truth... It is a "false" interpretation. Not easy to popularize


TP_LOCALLAB_WAT_SIGMALC_TOOLTIP;The effect of the local contrast adjustment is stronger for medium-contrast details and weaker for high and low-contrast details.\n This slider controls how quickly the effect dampens towards the extreme contrasts.\nThe higher the value of the slider, the wider the range of contrasts that will receive the full effect of the local contrast adjustment and the higher the risk of generating artifacts.\nThe lower the value, the more the effect will be pinpointed towards a narrow range of contrast values.